src/compat/posix/fs/dvfs/chmod.c

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file
+ * @brief
+ *
+ * @date 2.02.2016
+ * @author Alex Kalmuk
+ */
+
+#include <sys/stat.h>
+#include <errno.h>
+#include <fs/dvfs.h>
+
+int chmod(const char *path, mode_t mode) {
```

Comment:
I think it's better to check correctness of given "mode" arg.
Also I don't see where information is being updated on the media.